### PR TITLE
[Issue #421] Pillow recipe is broken

### PR DESCRIPTION
**Pull request by GoBig87**
This disables Little-CMS and allows it to build with out the missing symbols LCMS introduces.  It also sets the arch to arm64.  Before it wasn't building symbols for arm64.

### CODE REVIEW

File: recipes/pillow/__init__.py [hunk 1]
```diff
@@ -11,7 +11,7 @@ class PillowRecipe(Recipe):
     library = "libpillow.a"
     depends = ["hostpython3", "host_setuptools3", "freetype", "libjpeg", "python3", "ios"]
     pbx_libraries = ["libz", "libbz2"]
-    include_per_arch = True
+    archs = ['arm64']
```
Comment (Zen-CODE) [inline recipes/pillow/__init__.py:14]: Is this change necessary? What happens if you leave this as "include_per_arch"?

**Comment by AndreMiras** [comment]
Thanks @GoBig87 for the PR.
I wanted to merge your PR after fixing conflicts and addressing @Zen-CODE comments, but your remote seems gone. Are you still interested in pushing this contribution? Otherwise I can still push on your name, just cannot explicitly merge/ref this PR

**Comment by Zen-CODE** [comment]
@GoBig87 Thanks for this, but the pillow recipe is now fixed.
